Formula/julia.rb

```diff
@@ -0,0 +1,124 @@
+class Julia < Formula
+  desc "Fast, Dynamic Programming Language"
+  homepage "https://julialang.org/"
+  license all_of: ["MIT", "BSD-3-Clause", "Apache-2.0", "BSL-1.0"]
+  head "https://github.com/JuliaLang/julia.git"
+
+  stable do
+    url "https://github.com/JuliaLang/julia/releases/download/v1.6.1/julia-1.6.1.tar.gz"
+    sha256 "366b8090bd9b2f7817ce132170d569dfa3435d590a1fa5c3e2a75786bd5cdfd5"
+
+    depends_on arch: :x86_64
+  end
+
+  depends_on "cmake" => :build
+  depends_on "python@3.9" => :build
+  depends_on "curl"
+  depends_on "gcc" # for gfortran
+  depends_on "gmp"
+  depends_on "libgit2"
+  depends_on "libssh2"
+  depends_on "mbedtls"
+  depends_on "mpfr"
+  depends_on "nghttp2"
+  depends_on "openblas"
+  depends_on "openlibm"
+  depends_on "p7zip"
+  depends_on "pcre2"
+  depends_on "suite-sparse"
+
+  uses_from_macos "perl" => :build
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "patchelf" => :build
+    depends_on "libunwind"
+  end
+
+  def install
+    # Build documentation available at
+    # https://github.com/JuliaLang/julia/blob/v#{version}/doc/build/build.md
+    args = %W[
+      VERBOSE=1
+      USE_BINARYBUILDER=0
+      prefix=#{prefix}
+      USE_SYSTEM_CSL=1
+      USE_SYSTEM_PCRE=1
+      USE_SYSTEM_OPENLIBM=1
```

Comment:
What's the benefit of using `openlibm` over the system `libm` (`USE_SYSTEM_LIBM`)?

Reply:
`openlibm` is optimised for Julia (and vice-versa). `openlibm` is actually maintained by Julia maintainers. I don't have specific benchmarks, though.

Reply:
It also looks like Julia relies on some symbols that aren't necessarily in a given `libm` implementation: https://github.com/JuliaLang/julia/issues/26434#issuecomment-424585450

We could try system `libm`, but I suspect that's just a path to headaches.